src/repo/results.rs

```diff
@@ -0,0 +1,17 @@
+use libipld::cid::Cid;
+
+/// Describes the outcome of `BlockStore::remove`
+
+#[derive(Debug)]
+pub enum BlockRm {
+    Removed(Cid),
+    // TODO: DownloadCancelled(Cid, Duration),
+}
+
+// pub struct BlockNotFound(Cid);
+
+#[derive(Debug)]
+pub enum BlockRmError {
+    // TODO: Pinned(Cid),
+    NotFound(Cid),
+}
```

Comment:
I don't think it's a good idea to create a new file for this, there's plenty of room in the `src/repo/mod.rs`... In general I dislike grouping by type instead of feature. If that starts to create too large files, perhaps we have too many things which need to live next to each other and so our features need to become smaller :)